specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/examples/GetKeyPolicy-example.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "vaultBaseUrl": "https://myvault.vault.azure.net/",
+    "key-name": "key01",
+    "api-version": "7.3-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "https://myvault.vault.azure.net/keys/key01/policy",
+        "lifetimeActions": [
+          {
+            "trigger": {
+              "timeAfterCreation": "90D"
```

Comment:
Is the format of this duration intended to be [compliant with ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Durations)?